examples/llava/clip.cpp

```diff
@@ -1001,6 +1005,11 @@ struct clip_ctx * clip_model_load(const char * fname, const int verbosity = 1) {
     LOG_TEE("%s: CLIP using Metal backend\n", __func__);
 #endif
 
+#ifdef GGML_USE_CANN
+    new_clip->backend = ggml_backend_cann_init(0);
+    printf("%s: CLIP using CANN backend\n", __func__);
```

Comment:
```suggestion
    LOG_TEE("%s: CLIP using CANN backend\n", __func__);
```

Reply:
Thanks. I will go though new added code to avoid fprintf. Using LOG Macro instead.

Reply:
Done.